addons/account/models/account_invoice.py

```diff
@@ -1857,12 +1857,12 @@ def _prepare_invoice_line(self):
         }
         return data
 
-    @api.model
-    def create(self, vals):
-        if vals.get('display_type', self.default_get(['display_type'])['display_type']):
-            vals.update(price_unit=0, account_id=False, quantity=0)
-
-        return super(AccountInvoiceLine, self).create(vals)
+    @api.model_create_multi
```

Comment:
I asked myself the same and went to ask @rco-odoo before attempting this fix:

In v12+ all non `@api.model_create_multi` are implicitly `@api.model_create_single` [1] which accepts a simple dict or a list of dicts, in the case of the second though it will simply iterate over the recordset and call create one by one (no performance gain but it doesn't break) [2]

So yes, it is safe in stable :)

[1] https://github.com/odoo/odoo/blob/12.0/odoo/api.py#L320
[2] https://github.com/odoo/odoo/blob/12.0/odoo/api.py#L427-L433